python-package/xgboost/callback.py

```diff
@@ -588,36 +606,37 @@ class EvaluationMonitor(TrainingCallback):
     show_stdv : bool
         Used in cv to show standard deviation.  Users should not specify it.
     '''
-    def __init__(self, rank=0, period=1, show_stdv=False):
+    def __init__(self, rank=0, period=1, show_stdv=False) -> None:
```

Comment:
Yes if you want mypy to pass with `disallow_untyped_defs = True`.

Reply:
We should add mypy in the CI.

Reply:
@hcho3 Working on it, https://github.com/dmlc/xgboost/issues/6496  The effort is quite huge.  Getting dask interface up along is a lots of effort.  My current plan is to first make dask and related tests have type hints and run mypy on these files exclusively.   Then add hints to other places incrementally.

Reply:
What do you think?

Reply:
I can help adding mypy in the CI. It does not have to block merging for now. (CI should ignore failing mypy, for now)

Reply:
Thanks.  I will do that alone with the PR for typing dask interface.  The code becomes a bit more verbose, but I feel a lot safer and the data movement is clearer.  ;-)

Reply:
Can you share the command you're using to run `mypy`?

Reply:
A bit more changes, add:

```

[mypy]
ignore_missing_imports = True
disallow_untyped_defs = True
```
to `python-package/setup.cfg`, run `mypy .` under same directory.  This will get results from package but not test.